package/MDAnalysis/analysis/contacts.py

```diff
@@ -455,6 +462,10 @@ def __init__(self, u, select, refgroup, method="hard_cut", radius=4.5,
                                                 box=self._get_box(refA.universe)))
                 self.initial_contacts.append(contact_matrix(self.r0[-1], radius))
 
+        if len(self.initial_contacts) > 0:
+            self.n_initial_contacts = self.initial_contacts[0].sum()
```

Comment:
I think you pointed out that `self.initial_contacts` must always be longer than 0 -- do you still need the `if` here?

Reply:
Fair, i remove the ```if``` statement.